addons/docs/src/blocks/Source.tsx

```diff
@@ -74,11 +39,7 @@ export const getSourceProps = (
     source = targetIds
       .map(sourceId => {
         const data = storyStore.fromId(sourceId);
-        if (data && data.parameters) {
-          const { storySource } = data.parameters;
-          return storySource && extract(sourceId, storySource);
-        }
-        return '';
+        return data.parameters.docs?.source?.code || '';
```

Comment:
Let's go further and use [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)? =)

It won't change anything in this particular case (because `code` is a string anyway), it's just a sane default to use in pair with optional chaining, because the latter returns either null/undefined, or full chain result